### PR TITLE
Soap updates

### PR DESCRIPTION
**Pull request by afrasier**
- Update to match newest info re: DOS SOAP services

### CODE REVIEW

File: talentmap_api/bidding/urls/bid.py [hunk 1]
```diff
@@ -4,14 +4,16 @@
 from talentmap_api.bidding.views import bid as views
 
 urlpatterns = [
-    url(r'^(?P<pk>[0-9]+)/submit/$', views.BidListBidderActionView.as_view({'get': 'submit'}), name='bidding.Bid-bid-actions-submit'),
+    # RC-1 Disabled
```
Comment (burgwyn) [inline talentmap_api/bidding/urls/bid.py]: I think we want to exclude these changes from the `dev` branch

**Comment by afrasier** [comment]
Why did it merge RC-1 into this branch? It was supposed to go the other way.

**Comment by afrasier** [comment]
Eugh, I'll cherry pick the commits into a new branch tomorrow and make a new PR

**Comment by afrasier** [comment]
Making new PR with no RC-1 pollution

**Pull request closed without merging**